SampleGallery/Samples/SDK 16299/LinearGradients/LinearGradients.xaml.cs

```diff
@@ -0,0 +1,263 @@
+﻿//*********************************************************
+//
+// Copyright (c) Microsoft. All rights reserved.
+// This code is licensed under the MIT License (MIT).
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, 
+// INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, 
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. 
+// IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, 
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, 
+// TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH 
+// THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+//
+//*********************************************************
+
+using System;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Input;
+using Windows.UI.Composition;
+using Windows.UI.Xaml.Hosting;
+using Windows.UI;
+using System.Numerics;
+
+namespace CompositionSampleGallery
+{
+    public sealed partial class LinearGradients : SamplePage
+    {
+        public LinearGradients()
+        {
+            this.InitializeComponent();
+        }
+
+        public static string StaticSampleName => "Linear Gradients";
+        public override string SampleName => StaticSampleName;
+        public static string StaticSampleDescription => "Simple visual effect showing linear gradient animations. The first and last words will flip the colors from left to right.Clicking on any of the words will change the color scheme from warm to cool and back when clicked again." + Environment.NewLine + "This sample is to show how linear gradients can highlight content in colorful ways.";
```

Comment:
Break this line so it doesn't extend beyond 120 characters.
Also Environment.NewLine is not appropriate because that gives you the new line character for files on a platform - this isn't being used in a file. Whatever uses it will have its own notion of NewLine, and you should use that.

Reply:
i deleted the sentence and made it shorter.